src/info/infoPg.c

```diff
@@ -121,13 +121,13 @@ infoPgLoadCallback(void *data, const String *section, const String *key, const S
     if (strEq(section, INFO_SECTION_DB_STR))
     {
         if (strEq(key, INFO_KEY_DB_ID_STR))
-            loadData->currentId = jsonToUInt(value);
+            loadData->currentId = varUIntForce(value);
     }
     // Process db:history section
     else if (strEq(section, INFO_SECTION_DB_HISTORY_STR))
     {
         // Load JSON loadData into a KeyValue
```

Comment:
Comment should probably remove the word JSON since we're not loading the JSON to a kv anymore.

Reply:
Done.